### PR TITLE
Grammar Fix

### PR DESCRIPTION
**Pull request by dmartin4506**
Just repaired minor Grammar and Language Choice issues to improve legibility, no code altered

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/diagnostics.md [hunk 1]
```diff
@@ -41,6 +41,6 @@ If dramatically different output is caused by small changes, consider also
 adding them here.
 
 If you're using the stable version of the compiler, you should also check if the
-bug also exists in the beta or nightly versions. The output might also be
-different depending on the Edition.
+bug also exists in the beta or nightly versions. The output might differ depending
+upon the edition.
```
Comment (JohnTitor) [inline .github/ISSUE_TEMPLATE/diagnostics.md:45]: "also" here should remain IMO.

**Comment by JohnCSimon** [comment]
Ping from triage: 
@dmartin4506 - can you please address the comments

**Comment by crlf0710** [comment]
@dmartin4506 Ping from triage, any updates on this?

**Comment by crlf0710** [comment]
Triage: I'm gonna close this due to inactivity. Feel free to reopen or create a new PR when you've got time to work on this again. Thanks!

**Pull request closed without merging**